index.html

```diff
@@ -48,32 +48,199 @@ <h2>I'm a Software Developer</h2>
                     </li>
                     <li>
                         <a href="https://www.linkedin.com/company/vivimedia/about/" target="_blank" rel="noopener"> <img
                                 class="mysocial" src="/images/Vic.png" alt="" /></a>
                     </li>
                     <li>
-                        <a href="https://twitter.com/Engr_ImranKhan1" target="_blank" rel="noopener"> <img class="mysocial"
-                                src="/images/Twitter.png" alt="" /></a>
+                        <a href="https://twitter.com/Engr_ImranKhan1" target="_blank" rel="noopener"> <img
+                                class="mysocial" src="/images/Twitter.png" alt="" /></a>
                     </li>
                     <li>
-                        <a href="mailto:tipu870@gmail.com" target="_blank" rel="noopener"> <img class="mysocial" src="/images/Mail.png"
-                                alt="" /></a>
+                        <a href="mailto:tipu870@gmail.com" target="_blank" rel="noopener"> <img class="mysocial"
+                                src="/images/Mail.png" alt="" /></a>
                     </li>
                     ...
                 </ul>
             </div>
             <img class="bgimage" src="/images/Illustration.png" alt="" />
         </section>
+        <section class="myworks">
+            <h2 id="work-headline">My Recent Works</h2>
+            <img class="multi-story" src="/images/Multi-Post Stories.png" alt="">
+            <h2 class="work-subheading">Multi-Post Stories</h2>
+            <p class="work-specs">A daily selection of privately personalized reads; no accounts or
+                sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown
+                printer took a standard dummy text.</p>
+            <div class="for-flex">
+                <p class="flex-items item1">CSS</p>
+                <p class="flex-items item2">HTML</p>
+                <p class="flex-items item3">BootStrap</p>
+                <p class="flex-items item3">Ruby</p>
+            </div>
+        </div>
+        <button id="myworks-btn" class="work-btn">See Project</button>
 
-        
+        </section>
 
-    </main>
-    <footer>
+        <section class="projects-section">
+
+            <div id="proj1">
+                <h2 class="work-subheading projecttitle">Profesional Art Printing Data</h2>
+                <p class="work-specs projectdetails">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.</p>
+                <div class="for-flex projectflex2">
+                    <p class="flex-items item1 projectitem1">CSS</p>
+                    <p class="flex-items item2 projectitem2">HTML</p>
+                    <p class="flex-items item3 projectitem3">BootStrap</p>
+                </div>
+                <button class="work-btn projbtn">See Project</button>
```

Comment:
According to the requirements, only the parent container should be aligned with grid, hence i encourage you to use flex to align contents within each card to make it easier to work with.